mne/viz/topo.py

```diff
@@ -713,22 +706,28 @@ def _plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
 
     if not merge_channels:
         # XXX. at the moment we are committed to 1- / 2-sensor-types layouts
-        chs_in_layout = set(layout.names) & set(ch_names)
-        types_used = {channel_type(info, ch_names.index(ch))
-                      for ch in chs_in_layout}
+        chs_in_layout = [ch_name for ch_name in ch_names
+                         if ch_name in layout.names]
+        # Using List comprehension to order elements and prevent randomness
```

Comment:
```suggestion
```

I would just remove this comment. A new contributor will not know what you mean by "randomness"

Reply:
Added suggestion to https://github.com/mne-tools/mne-python/pull/9207/commits/744da2bd53cabd84b4f1fde9246271b15428f3c3